src/main.zig

```diff
@@ -1718,8 +1718,11 @@ fn buildOutputType(
                         try cc_argv.append(arena, arg);
                     } else if (mem.startsWith(u8, arg, "-I")) {
                         try cssan.addIncludePath(arena, &cc_argv, .I, arg, arg[2..], true);
-                    } else if (mem.eql(u8, arg, "-x")) {
-                        const lang = args_iter.nextOrFatal();
+                    } else if (mem.startsWith(u8, arg, "-x")) {
+                        const lang = if (arg.len == 2)
```

Comment:
```suggestion
                        const lang = if (arg.len == "-x".len)
```

Bit of a nitpick, but just to be consistent with the slice operation below.